src/vstart.sh

```diff
@@ -1314,6 +1315,11 @@ else
         debug ms = 1'
 fi
 
+if [ "$ceph_osd" == "crimson-osd" ]; then
```

Comment:
Might be worth leaving a comment explaining we don't support PG merge / split yet.